packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -1030,6 +1032,8 @@
   "Encrypted": "Encrypted",
   "Encrypted_channel_Description": "End to end encrypted channel. Search will not work with encrypted channels and notifications may not show the messages content.",
   "Encrypted_message": "Encrypted message",
+  "EncryptionKey_Change_Disabled": "You can't change your encryption key because it was never used on this browser.",
```

Comment:
I'd suggest:
`You can't set a password for your encryption key because your private key is not present on this client. Please go to the client you have the encryption working and set the password there to be able to import it here.`